resources/views/campaigns/grouptype-select.blade.php

```diff
@@ -0,0 +1,12 @@
+<div class="select">
+    <select name="{{ $name }}">
+        <option value="" {{ ! old($name) && empty($value) ? 'selected' : '' }} disabled>{{ $placeholder or '–' }}</option>
+        @foreach($options as $option)
+            @if (old($name))
+                <option value="{{ $option->id}}" {{ old($value) === $option->id? 'selected': '' }}>{{ $option->name }}</option>
+            @else
+                <option value="{{ $option->id }}" {{ isset($value) && $value === $option->id ? 'selected': '' }}>{{ $option->name  }}</option>
+            @endif
+        @endforeach
+    </select>
+</div>
```

Comment:
This is arguably _more_ work, so just throwing the thought out there: I wonder if we could map over the `GroupType`s in the controller to obtain a similar key-value pairing that we have on other selections like this (e.g. the [`PostType::labels`](https://github.com/DoSomething/rogue/blob/01124649b964cd157f116ae0d2419934b27b5ca7/app/Types/PostType.php#L19-L31)) so that we could just use the existing [forms.select.blade.php](https://github.com/DoSomething/rogue/blob/01124649b964cd157f116ae0d2419934b27b5ca7/resources/views/forms/select.blade.php#L1) form? 

e.g. (rough idea offhand) 
```php
GroupTypes::all()->map(function($groupType) {
  return [$groupType->id => $groupType->name];
)->flatten();
```

And this could even be a new `GroupType::labels()` convenience method on the model 🤔

Reply:
Using the existing `forms.select.blade` would be ideal - great suggestion @mendelB !

Reply:
This file should be removed per latest changes.